Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,39 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
+
+            Program.HomeworkOne();
+            //Program.HomeworkOneB("Stefan", "Andersson", 41f, 95f, 178f);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069627)